consumer/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/consumer/service/saker/SakerServiceImpl.java

```diff
@@ -113,8 +109,11 @@ private String opprettSak(String fnr, Sak sak) {
                             .withSakstype(new WSSakstyper().withValue(sak.sakstype)));
 
             return behandleSakWS.opprettSak(request).getSakId();
-        } catch (OpprettSakUgyldigInput | OpprettSakSakEksistererAllerede e) {
-            throw new RuntimeException(e);
+        } catch (OpprettSakSakEksistererAllerede opprettSakException) {
+            return finnSakIdFraGsak(fnr, sak)
```

Comment:
Kan man vel flytte den sjekken før prøve å opprette saken ?

Reply:
Blir ett spørsmål om hva vi optimaliserer for tenker jeg.

Vi kaller `opprettSak` når både `finnesIGsak` og `finnesIPsak` er `false`, og i de fleste tilfeller vil det være korrekt å gjøre, men så har vi noen få tilfeller hvor saken allerede har blitt opprett. 

Med løsningen som først prøver å opprette, og deretter håndterer eventuelle feil, så vil vi stort sett gjøre det "riktige" samtidig som vi håndterer de litt mer spesielle situasjonene. 
Om vi bytter om, gjør `finnSakIdFraGsak` og deretter `opprettOppgave` om det er nødvendig, så vil vi det bli så vi først håndterer spesielttilfellene, og deretter gjør det vi egentlig tror er nødvendig. Mistenker at trafikken mot gsak da kommer til å øke vesentlig. :/

Reply:
ok...